package-lock.json

```diff
@@ -21,9 +21,6 @@
         "lint-staged": "^14.0.1",
         "prettier": "3.0.2",
         "typescript": "^5.0.4"
-      },
-      "engines": {
```

Comment:
This removes the `engines` section from this file. Is this a manual edit? Or why is this removed now?

Reply:
Sorry, this was a mistaken commit.

> This removes the `engines` section from this file. Is this a manual edit? Or why is this removed now?